assets/ui.css

```diff
@@ -427,6 +427,27 @@
   background: url("https://respec.org/xref/loader.gif") no-repeat center;
 }
 
+.respec-dfn-list li span {
```

Comment:
I would prefer using a proper classname here rather than `li span`, for example `.dfn-status`.

Reply:
@saschanaz makes sense. in case we do have other elements that also match `li span`. added `.dfn-status` in https://github.com/w3c/respec/pull/2711/commits/d5108429caa30bd82e42cc1852b33a35b036fa53!

Reply:
@saschanaz makes sense. in case we do have other elements that also match li span. added .dfn-status in d510842!